openqa-investigate

```diff
@@ -374,7 +374,14 @@ investigate() {
     comment_id=$(sync_via_investigation_comment "$id" "$first_cluster_job_id") || rc=$?
     [[ $rc != 255 ]] && return $rc
 
-    out=$(trigger_jobs "$id" "${@:2}") || return $?
+    out=""
+    if ! out=$(trigger_jobs "$id" "${@:2}"); then
+        local comment="Triggering investigation jobs failed.
+        Check journal for reasons."
```

Comment:
Doesn't `$out` contain the reasons that the reader might be interested in? So how about something like

```suggestion
        local comment="Triggering investigation jobs failed: $out"
```

Reply:
$out is empty if it pass the if statement, why print an empty variable then?

Reply:
I see. Is there details in the stderr that we don't catch?

Reply:
Change with including stderr in the openqa comment!